### DISCUSSION TITLE
Looking for Feedback / Ideas for an upcoming update

### DISCUSSION THREAD

**Discussion by bcs4313** [bcs4313/Artifacts-Of-DOOM#5] [Ideas]
I have decided to continue development with this mod as of now.

Here you can post any of your artifact ideas and/or modifications to existing artifacts.
- If you post to this thread, I will discuss it with you and make changes if I find your suggestion to be fun and/or helpful to the pack.
- This includes entropy effects and events. I love adding extra outcomes / events to that one!

~These changes will be applied for version 1.6. 1.6 will add a new artifact as well.

**Comment by ElyjahWirth** [comment]
Hey there! My husband and I love your mod pack, it has breathed new life into one of our favorite games. We do almost every run with Artifact of War. We found a bug with the modifier that only impacts clients. It doesn't impact Hosts or Singleplayer. The issue is that the client's healthbars (for themselves, the monsters, and allies) all get wonky. In late game, this results in ally health bars going across the whole screen.

Would super appreciate any help with this bug. Let me know if you need any more info or logs or anything. Thank you!

![20230722115857_1](https://github.com/bcs4313/Artifacts-Of-DOOM/assets/35857898/6639decb-77a2-46fb-b0a6-5c172b4b63da)
![20230722114229_1](https://github.com/bcs4313/Artifacts-Of-DOOM/assets/35857898/21331d76-1ada-4a2d-9441-94efd600b421)

**Comment by bcs4313** [comment]
Hello!

Thank you for the report. That is a pretty obnoxious bug to have. I'll take a look at it and see what I can do to fix it today.

**Comment by bcs4313** [comment]
I see what's happening now. So if there is an improper alignment with actual health and max health, it causes an overflow. The overflow creates the weird bar you are talking about. So I need to create a network packet to update the max health of entities on the client side. When its done, the bug will go away.

**Comment by bcs4313** [comment]
Alright, the bug is fixed as of now. I will be releasing this fix with version 1.6.

**Comment by bcs4313** [comment]
1.6 is now out to download.
